paperqa/utils.py

```diff
@@ -184,4 +184,12 @@ def llm_read_json(text: str) -> dict:
     text = "{" + text.split("{", 1)[-1]
     # split anything after the last }
     text = text.rsplit("}", 1)[0] + "}"
+
+    # escape new lines within strings
```

Comment:
What do you think of [`re.escape`](https://docs.python.org/3/library/re.html#re.escape)?

Reply:
Don't see the connection